addons/payment_nuvei/tests/common.py

```diff
@@ -0,0 +1,50 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.payment.tests.http_common import PaymentHttpCommon
+
+
+class NuveiCommon(PaymentHttpCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.maxDiff = None
+        cls.nuvei = cls._prepare_provider('nuvei', update_values={
+            'nuvei_merchant_identifier': '123456abc',
+            'nuvei_site_identifier': '1234',
+            'nuvei_secret_key': 'dummy',
+        })
+
+        cls.provider = cls.nuvei
+
+        cls.notification_data = {
+            "ppp_status": "OK",
+            "LifeCycleId": "7110000000004858226",
```

Comment:
You can only set the values that are necessary. We never check this `LifeCycleId` for instance, so no need to add it in the test setup